saleor/graphql/schema.graphql

```diff
@@ -11834,6 +11843,77 @@ enum CollectionSortField @doc(category: "Products") {
   PUBLISHED_AT
 }
 
+input ProductWhereInput @doc(category: "Products") {
+  metadata: [MetadataFilter!]
+  id: [ID!]
+  name: StringFilterInput
+  slug: StringFilterInput
+  productType: GlobalIDFilterInput
+  category: GlobalIDFilterInput
+  collection: GlobalIDFilterInput
+  isAvailable: Boolean
+  isPublished: Boolean
+  isVisibleInListing: Boolean
+  publishedFrom: DateTime
+  availableFrom: DateTime
+  hasCategory: Boolean
+  price: FloatFilterInput
+
+  """Filter by the lowest variant price after discounts."""
+  minimalPrice: FloatFilterInput
+  attributes: [AttributeInput!]
+
+  """Filter by variants having specific stock status."""
+  stockAvailability: StockAvailability
+  stocks: ProductStockFilterInput
+
+  """Filter on whether product is a gift card or not."""
+  giftCard: Boolean
+  hasPreorderedVariants: Boolean
+
+  """Filter by when was the most recent update."""
+  updatedAt: DateTimeRangeInput
+
+  """List of conditions that must be met."""
+  AND: [ProductWhereInput!]
+
+  """A list of conditions of which at least one must be met."""
+  OR: [ProductWhereInput!]
+}
+
+"""
+Define the filtering options for foreign key fields.
+
+Added in Saleor 3.14.
+
+Note: this API is currently in Feature Preview and can be subject to changes at later point.
+"""
+input GlobalIDFilterInput {
+  """The value equal to."""
+  eq: ID
+
+  """The value included in."""
+  oneOf: [ID!]
+}
+
+"""
+Define the filtering options for integer fields.
+
+Added in Saleor 3.11.
+
+Note: this API is currently in Feature Preview and can be subject to changes at later point.
+"""
+input FloatFilterInput {
+  """The value equal to."""
+  eq: Float
+
+  """The value included in."""
+  oneOf: [Float!]
+
+  """The value in range."""
+  range: PriceRangeInput
```

Comment:
Shouldnt we have `FloatRangeInput` or `RangeInput` here? `PriceRangeInput` could be missleading, as the input has a name `FloatFilterInput`

Reply:
Also, shouldn't we use here `Decimal` scalar instead of `Float`?  `Decimal` could be a better choice, as we use it in all price fields available in API.